hardware/amlogic/lafrite.md: Fix broken link

leon-anavi opened the pull request:
Fix the broken link to the U-Boot firmware files.

chewitt commented:
@leon-anavi It looks there is no .scr file in the updated share. Can you revise that part of the instructions too?

leon-anavi commented:
> @leon-anavi It looks like there is no .scr file in the updated share. Can you revise that part of the instructions too?

Yes, I updated them. Now it is needed just to flash `aml-s805x-ac-spiflash.img` on a USB stick. So as an outcome I removed one of the steps.

chewitt reviewed (approved):
Thank you! 👍🏻
